docs/proposals/ReID-example-for-Sedna.md

```diff
@@ -0,0 +1,88 @@
+
+Now, the architecture of the Re-ID feature is as below (see in [Re-ID Tutorial](https://github.com/vcozzolino/sedna/blob/feature-reid-ct/examples/multiedgetracking/tutorial/tutorial.md)):
+
+<img src=".\images\ReID-example-for-Sedna-ReIDarch.png" style="zoom:75%;" />
+
+Based on the architecture, this proposal is to develop an end-to-end application based on the Sedna's Re-ID feature. Under the scenario of Re-ID inference, build a user-friendly end-to-end application by taking advantage of cloud-edge collaboration. This may provide some references for people who want to use the Re-ID feature in different scenarios.
+
+### Goals
+
+- Based on the Re-ID feature, develop an application for video human flow statistics and security authentication.
+  - Video sources from specific opensource datasets can be PRID2011, etc.
+  - Video sources from multiple angles/devices are selected as different edge Pods to give full play to the multi-edge collaboration capability of KubeEdge.
+  - Count the human flow at the edge side, obtain the human flow statistics on the cloud (for demonstration, analysis or prediction). In addition, the cloud side also has authentication capabilities for security purposes using the Re-ID feature.
+  - Provide a user-friendly results visualization website/App using `Re-ID Manager`'s API for video streaming and other functions. 
+- Extend more examples and scenarios for Sedna and its new feature -- `Multi-edge tracking Re-ID`.
+  - Provide some references for developers who are interested in Sedna.
+  - Provide more examples for Sedna's official documentation. 
+
+## Proposal
+
+We propose building an application for video human flow statistics and security authentication which can be used in industrial park, apartment and other similar scenarios. 
+
+Human flow statistics are calculated at the edge side, and the data of the human flow can be used for site management, reasonable allocation of public resources, anti-stampede warning and so on. The cloud side records the temporal changes of the human flow (can be used to predict the human flow in the future), and the Re-ID module is used for security authentication to identify suspicious characters.
+
+### Use Cases
+
+- Users can query human flow statistics from the cloud side.
+- Users can use the history human flow data recorded on the cloud to predict human flow in the future.
+- Users can query information about suspicious character through the records in the database.
+- The system can help administrators identify the name/ID of people whose information has already been stored in the database.
+
+## Design Details
+
+### Architecture and Modules
```

Comment:
The design should be more detailed, such as which DB will be used for your system.